tmscti/callsupply.sqf

```diff
@@ -104,4 +104,25 @@ if (_transportcraft == "C-17 Globemaster III") then {
             _wp0 setWaypointStatements ["true", "cleanUpveh = vehicle leader this; {deleteVehicle _x} forEach crew cleanUpveh + [cleanUpveh];"];
             }];
        }];
-    };
\ No newline at end of file
+    };
+	
+if (_transportcraft == "Self Delivery (Jet)") then {
+    
+	_supply_item_data_string = lbData [1507, 0];
+    _supply_item_data = call compile _supply_item_data_string;
+	_jet_classname = _supply_item_data select 1;
+	_jet = CreateVehicle [_jet_classname, getMarkerPos "sp_e", [], 0, "FLY"];
+	createVehicleCrew (_jet);
+	
+	_groupjet = group _jet;
+	_groupjet setBehaviour "CARELESS";
+
+    _jet landAt 5;
+
+    [[(driver _jet), "Aircraft approaching sand airfield, clear runway!"], "sideChat", west, false, false] call BIS_fnc_MP;
```

Comment:
Two problems with this line:

- Side west is hardcoded, even though this probably should work for side east as well
- It doesn't seem to actually work.